packages/angular-display/lax/README.md

```diff
@@ -0,0 +1,6 @@
+# @cloudflight/angular-display/lax
```

Comment:
What is the usecase of providing this option?

Reply:
projects that for whatever reason have problems configuring the types correctly. Those can still use the library but of course have no type safety. But the previous version didn't even offer the type safe option.

Reply:
Is that a hypothetical usecase you have in mind or do we have real life projects with this issue?

Reply:
a bit of both.

It was not extremely straight forward to setup the additional types for my current project. Which is not yet big and very complex, but is split up into multiple angular libraries using the Angular CLI. Each of those libraries has their own tsconfig which extends from a global one.   
I added the file containing the necessary file extension to the include list of the global tsconfig, but one tsconfig also specified a include list of their own and therefor overwriting the global include list (since tsconfig values are not merged but overwritten). This caused the IDE complaining in one library while the compilation of the whole app worked fine.   
In the end I found and solved this.

But I can see projects where resolving something like that might not be as straight forward. That's why I included it. It was easy to do and does not require any additional maintenance. Its just a second export that also defines the Breakpoints to not be type-safe.   
I have not tested it, but in theory this should even allow you to use both versions of the library in the same application without blowing up the bundle or any other oddities, since its just a reexport of the same thing.